packages/web/src/scss/components/Tooltip/index.html

```diff
@@ -7,7 +7,7 @@ <h2 class="docs-Heading">Placements</h2>
     <div class="docs-Stack docs-Stack--start">
 
       <form autocomplete="off" class="mx-auto">
-        <div class="Grid Grid--cols-3" style="align-items: center; justify-items: center">
+        <div class="Grid Grid--cols-3" style="align-items: center; justify-items: center; gap: 80px;">
```

Comment:
I know this is just a simple workaround but the tooltip popover shows over a radio button when using `rightBottom`.